cli/compiler/AbstractCompiler.js

```diff
@@ -0,0 +1,62 @@
+import childProcess from 'child_process';
+import path from 'path';
+
+class Compiler {
```

Comment:
Thanks missed that. I think I'm just gonna change those into a bash script for readability, those js Classes were useful in my other projects where I use that for different environments.